### ISSUE TITLE
Cluster email flow (for single party deliveries only)

### ISSUE THREAD

**Issue by maybejustmaybe** [bedstuystrong/automation_prime#35]
One small step for automation, one giant leap for de-Slacking... This new email flow will be the primary way delivery volunteers interact with the automation. Hopefully, this will significantly lower the technological barrier to participating in our delivery operation.

For this task, let's try and stand up an MVP that can be used for _single party deliveries_, where one volunteer is responsible for both shopping and driving. This seems to be the immediate need of the system, not more complex multi party deliveries (e.g. with a different shopper and driver). However, let's do our best to make a system that could easily be adapted for multi-party deliveries.

Components:
- [ ] Add a `Sent Email?` checkbox field to the clusters table
- [ ] Send the email in the `cluster.py::onStatusAssigned` function
- [ ] Partner up with food ops to figure out how to structure the email (demos encouraged)

Email Features (suggested):
- The reply-to of the email should include the member + operations@bedstuystrong.com
- HTML email is generated from cluster model using Jinja template
- Improved shopping list rendering (may require item lookup table cleanup)

See how we currently generate the email and shopping list: [click here](https://github.com/bedstuystrong/firebase-functions/blob/ea7497e177dbcb3c79d15826e9e4ee847ed92f9f/functions/slackEvents.js#L209)

depends on #36

**Comment by maybejustmaybe** [comment]
How do members communicate where they are going to meet up? Is this something the intake coordinator could take care of (e.g. pick the grocery store)?

**Comment by maybejustmaybe** [comment]
How/when do we confirm neighbor availability for those time slots?

**Comment by maybejustmaybe** [comment]
Example of current email that operations sends out for weekend deliveries:

```
Title: BSS Sunday 2/21 grocery delivery / neighbor info & shopping list

Hi James, 

This is your email with contact information and delivery instructions for A, ticket ID# 1, B, ticket ID# 2 and C, ticket ID# 3     

# NOTE that Rachel wrote this part
We would be thrilled to send you a fourth ticket per your sign-up, but two of these are larger households, and we wanted to keep this manageable for you - let me know if you'd like another ticket anyway (or not is totally cool)

We’ve included a step-by-step checklist, plus the complete grocery list(s) below, along with spending guidance.

Please note that when you call to coordinate your delivery time if a neighbor asks to add additional items and they already have 3 custom items on their list please let them know that you can't make any additions.
 
Thanks so much for being here, and please reach out with any questions! 

Rachel

Bed-Stuy Strong

**********

First, call your neighbors. Please call when you get this email, or shortly after - your neighbor will be expecting to hear from you! Agree on a good delivery time for the weekend, confirm their address, and ask any questions about their grocery list (ie if “baby formula” do they have a preferred brand?) 

**If you can’t get a hold of someone, please try back a few times, leave a message, and text. If there’s still no response, let us know at operations@bedstuystrong.com.

Shop the grocery list. Keep yourself safe - wear a mask, keep distance when possible, and wash your hands when you get home from shopping. Use the list as your guide, and do your best to purchase what your neighbor asked for. Be mindful of household size to determine quantities; you are looking to get a week's worth of food for your neighbor.  Make sure to take a photo of your receipt for the completion form!

Safely deliver the groceries. Call/buzz when you arrive, then back up 6 feet for COVID safety. Please stick around to say hello and make sure they receive the bags. ❤ This is about being in relationship to our neighbors 😊  Once you’ve dropped off groceries, check in after! A simple text to check in / ask how they’re doing is always appreciated. ❤

If you can, please snap pictures of your volunteering efforts for our communication outputs (social, newsletter, etc.)! You can email them directly to us, we’d greatly appreciate it!

Fill out the completion form to let us know your delivery is complete, and to receive your reimbursement within 48 – 72 hours.

A note: Some neighbors will offer or ask about making contributions to offset their grocery delivery. Thank them and accept their contribution; anything they wish to donate goes right back into more groceries for others.

** If the neighbor hands you cash, please Venmo BSS for the amount. If you do not have access to Venmo, email us and we will figure out another way to accept the contribution!

** If the neighbor is younger / tech savvy, or notes that they would like to donate in the future, feel free to direct them to www.bedstuystrong.com for information on how to donate (or share BSS Venmo/Cashapp info)

If you have any issues along the way, please reach out to us at operations@bedstuystrong.com and someone will get back to you. 

Neighbor Information

Ticket ID: 1
Neighbor: B (<intersection>)
Address: <address>
Phone: <number>
Delivery Notes: null
Vulnerabilities: null
Household Size: <size>

Fridge / Frozen:
 2 1/2 gallon Lactaid
 2 A dozen Eggs
 1 1 package Butter
Non-perishable:
 2 1 lbs Bag White Rice
 3 1 Loaf Whole Wheat Bread
 3 1 Loaf White Bread
 2 1 box Cereal (Frosted Flakes)
 2 1 gallon jug Apple Juice
 2 1 gallon jug Orange Juice
 4 1 can Canned Mixed Vegetables
 8 1 can Canned Tuna
 
Produce:
 3 A bunch Spinach
 3 One bunch Bananas
 7 One orange Oranges
 8 One apple Apples
 2 A bag Potatoes
 2 A bag Onions
Other Items: Tide laundry detergent; non-alcoholic cleaning wipes, paper towels, toilet paper, water bottles

Spending guidance:
- $350 for a large household (6+ ppl)
- Please try to buy about a week’s worth of food for the household. It’s ok if you can’t get every single thing on the shopping list--the main goal is that the family’s nutritional needs are sufficiently met.

**********************

<more tickets>
```

**Comment by leifwalsh** [comment]
Fixed by #44
